prime-router/settings/organizations-prod.yml

```diff
@@ -446,6 +446,25 @@
         type: REDOX
         apiKey: 9e7f9bd1-a876-4f10-9d17-0833dc7bf60d # Redox prod
 
+    - name: elr-philadelphia-prod
```

Comment:
I don't think its necessary to keep organizations-prod.yml up to date any more.  Mainly we need a -local, and it only needs to include test orgs/receviers/senders.